pkg/portal/graphql/query.go

```diff
@@ -109,5 +111,95 @@ var query = graphql.NewObject(graphql.ObjectConfig{
 				}).Value, nil
 			},
 		},
+		"activeUserChart": &graphql.Field{
+			Description: "Active users chart dataset",
+			Type:        activeUserChart,
+			Args: graphql.FieldConfigArgument{
+				"appID": &graphql.ArgumentConfig{
+					Type:        graphql.NewNonNull(graphql.String),
+					Description: "Target app ID.",
+				},
+				"periodical": &graphql.ArgumentConfig{
+					Type: graphql.NewNonNull(periodicalEnum),
+				},
+				"rangeFrom": &graphql.ArgumentConfig{
+					Type: graphql.NewNonNull(graphqlutil.Date),
+				},
+				"rangeTo": &graphql.ArgumentConfig{
+					Type: graphql.NewNonNull(graphqlutil.Date),
+				},
+			},
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				appID := p.Args["appID"].(string)
```

Comment:
To stay consistent, the appID we accept here should be encoded app ID. That is base64 encoded `App:raw_app_id`.